fsspec/implementations/local.py

```diff
@@ -128,6 +128,9 @@ def mv_file(self, path1, path2, **kwargs):
         path2 = self._strip_protocol(path2).rstrip("/")
         shutil.move(path1, path2)
 
+    def rm_file(self, path):
+        self.rm(path)
```

Comment:
This ends up being sort of circular. I know that for this implementation, the actual removal is in `rm`, but that's not the normal pattern, so should we just use `os.remove` directly here? It'll make the right kind of exception if the path is a directory.

Reply:
This implementation does raise the IsADirectory error if the path is a directory (it falls through to the os.remove, since recursive is false).  I don't think it's truly circular -- it's just been implemented the opposite of the other classes.

But you've made me realize an actual problem -- this implementation accepts a list when it shouldn't.

I've updated to your suggestion, which is correct.  I also added a couple more checks in the test.